### ISSUE TITLE
Legend icons are really small compared to the data points within the graph

### ISSUE THREAD

**Issue by JananiGunasekaran** [cerner/terra-graphs#228]
# Bug Report

## Description

We are consuming terra-graphs in our application. We see that the legend icons are really small compared to the icons within the graph. Users are unable to distinguish or recognise the legends.

### Graphs affected

* [ ]  Bar graph
* [ ]  Bubble graph
* [ ]  Gantt chart
* [x]  Line graph
* [x]  Paired result
* [ ]  Pie chart
* [ ]  Scatter plot
* [ ]  Timeline

## Steps to Reproduce

1.  The issue can be seen here: https://pages.github.cerner.com/assessments/advanced-graphing-js/tests/advanced-graphing-js/application-per-tab

## Link to CodeSandbox
Use [this](https://codesandbox.io/s/keen-thunder-nh5gk?file=/src/data.js) template to reproduce the behavior.

## Additional Context / Screenshots

![image](https://user-images.githubusercontent.com/60426528/131958319-eeb187db-76fc-43eb-b9ed-87042404421e.png)

![image](https://user-images.githubusercontent.com/60426528/131957093-ee11ed55-a64b-487d-bdf1-2a0fc969db9a.png)

## Expected Behavior

Legend icons should have same size as that of the data points within the graph

## Environment

* Component Name and Version: Assessments-graph-js & Advanced-graphing-js
* Browser Name and Version: Google Chrome 92.0.4515.131
* Node/npm Version: [e.g. Node 8/npm 5] Node 10.16.2 / npm 6.14.6
* Webpack Version:
* Operating System and version (desktop or mobile): mac0S

## @ Mentions

@AshishMotanamGurunadham

**Comment by MadhavaNaresh** [comment]
@AshishMotanamGurunadham Any update on this issue..?

**Comment by MadhavaNaresh** [comment]
@AshishMotanamGurunadham , @benbcai @xenoworf Can someone look into and provide an update on this issue, its a defect in our backlog for quite sometime now.

**Comment by xenoworf** [comment]
@chrismichalewicz FYI this request. Let's discuss it at our next meeting.
@MadhavaNaresh sorry for this slipping though the cracks. I will try to get this discussed soon. I'll post back next week no matter what so you know we have not forgotten about it.

**Comment by chrismichalewicz** [comment]
I have created an item in our internal backlog so that we can include this in our list of items to be considered for prioritorization.  At this point I do not have a time frame for when we'd be able to start work on this issue.

**Comment by MadhavaNaresh** [comment]
@chrismichalewicz Can you share the jira you created here so we can link them, below is the defect on our graphing product.
https://jira2.cerner.com/browse/ADVGRAPH-2193

**Comment by chrismichalewicz** [comment]
@MadhavaNaresh  I apologize the internal Jira we have for this is https://jira2.cerner.com/browse/UXPLATFORM-6419 which looks like you or someone from your team already associated it.  At the moment we do not have it prioritized but will continue to include it in our upcoming PI plannings to ensure it gets considered for prioritization.
